### DISCUSSION TITLE
It doesn't seem to work properly when deployed on Baidu Cloud.

### DISCUSSION THREAD

**Discussion by Initial-heart-1** [anuraghazra/github-readme-stats#2341] [Q&A] [answered]
I am in China. Due to some features of China, it is difficult for us to access Vercel, so I have tried to find an alternative located in China. 
Now, I chose Baidu Cloud, which does not allow me to set the location of index.js, it must be in the root directory of the uploaded ZIP.
So, I moved the entire src folder to the root directory.
I removed some files due to the size limit of the uploaded files. Then I configured the environment variables, but they don't seem to work.

<details>
<summary>Dir tree</summary>

> No node_modules

github-readme-stats
├─ api
│    ├─ index.js
│    ├─ pin.js
│    ├─ top-langs.js
│    └─ wakatime.js
├─ calculateRank.js
├─ cards
│    ├─ index.js
│    ├─ repo-card.js
│    ├─ stats-card.js
│    ├─ top-languages-card.js
│    ├─ types.d.ts
│    └─ wakatime-card.js
├─ codecov.yml
├─ common
│    ├─ Card.js
│    ├─ I18n.js
│    ├─ blacklist.js
│    ├─ createProgressNode.js
│    ├─ icons.js
│    ├─ index.js
│    ├─ languageColors.json
│    ├─ retryer.js
│    └─ utils.js
├─ fetchers
│    ├─ repo-fetcher.js
│    ├─ stats-fetcher.js
│    ├─ top-languages-fetcher.js
│    ├─ types.d.ts
│    └─ wakatime-fetcher.js
├─ getStyles.js
├─ index.js
├─ node_modules
├─ package-lock.json
├─ package.json
├─ scripts
│    ├─ close-stale-theme-prs.js
│    ├─ generate-langs-json.js
│    ├─ generate-theme-doc.js
│    ├─ helpers.js
│    └─ preview-theme.js
├─ themes
│    └─ index.js
└─ translations.js
</details>
<details>
<summary>Log</summary>

Return(Browser):
```
{"errorMessage":"Unexpected token export"}
```

Return(Full):
```
{"errorMessage":"Unexpected token export","errorType":"SyntaxError","stackTrace":["export * from \"./common/index.js\";","^^^^^^","","SyntaxError: Unexpected token export","at Module._compile (internal/modules/cjs/loader.js:703:23)","at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)","at Module.load (internal/modules/cjs/loader.js:628:32)","at Function.Module._load (internal/modules/cjs/loader.js:555:12)","at Module.require (internal/modules/cjs/loader.js:666:19)","at require (internal/modules/cjs/helpers.js:16:16)","at Interface.emit (events.js:196:13)","at Interface._onLine (readline.js:314:10)"]}
```

Console:
```
{"errorMessage":"Unexpected token export","errorType":"SyntaxError","stackTrace":["export * from \"./common/index.js\";","^^^^^^","","SyntaxError: Unexpected token export","at Module._compile (internal/modules/cjs/loader.js:703:23)","at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)","at Module.load (internal/modules/cjs/loader.js:628:32)","at Function.Module._load (internal/modules/cjs/loader.js:555:12)","at Module.require (internal/modules/cjs/loader.js:666:19)","at require (internal/modules/cjs/helpers.js:16:16)","at Interface.emit (events.js:196:13)","at Interface._onLine (readline.js:314:10)"]}
```

Full log matches the output of Console log DM.
</details>

**Comment by rickstaa** [comment, marked as answer]
@Initial-heart-1 We currently do not support other deployments than https://vercel.com/. However, feel free to create a PR that adds Baidu cloud support. 👍

**Reply by Initial-heart-1** [reply]
@rickstaa Thanks for your answer. I'd like to do this but I haven't had a lot of time recently and there are some problems that have not allowed me to do this, for which I am very sorry.

**Reply by rickstaa** [reply]
@Initial-heart-1 I think you can use https://github.com/anuraghazra/github-readme-stats/pull/2447 to deploy GRS on Baidu. 👍

**Reply by Initial-heart-1** [reply]
@rickstaa I viewed this pull request. I think it may resolve my problem. I will try it out as soon as possible. Thank you.

**Reply by Omnivo-7** [reply]
Yes i think
